exchanges/kucoin/kucoin_test.go

```diff
@@ -1090,9 +1087,42 @@ func TestGetTradingFee(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
 
-	_, err := ku.GetTradingFee(context.Background(), "BTC-USDT")
+	_, err := ku.GetTradingFee(context.Background(), nil)
```

Comment:
For this specific test, you can place it before the `SkipTestIfCredentialsUnset` check to enhance offline test coverage